rp2040-hal/Cargo.toml

```diff
@@ -4,10 +4,12 @@ version = "0.9.1"
 authors = ["The rp-rs Developers"]
 edition = "2021"
 homepage = "https://github.com/rp-rs/rp-hal"
-description = "A Rust Embeded-HAL impl for the rp2040 microcontroller"
+description = "A Rust Embedded-HAL impl for the rp2040 microcontroller"
 license = "MIT OR Apache-2.0"
 rust-version = "1.75"
 repository = "https://github.com/rp-rs/rp-hal"
+categories = ["embedded", "hardware-support", "no-std", "no-std::no-alloc"]
+keywords = ["embedded", "hal", "raspberry-pi", "rp2040", "embedded-hal"]
```

Comment:
Not sure about "raspberry-pi", as this usually means the raspberry single board computers. But as `embassy-rp` also includes this keyword, there's at least some precedent.

Reply:
yeah, i actually took the keywords from embassy (minus `async`)